src/main/java/com/couchbase/lite/store/SQLiteStore.java

```diff
@@ -2141,14 +2141,12 @@ protected boolean endTransaction(boolean commit) {
      * would require an expensive full tree traversal. Hopefully this way is good enough.
      */
     protected int pruneRevsToMaxDepth(int maxDepth) throws CouchbaseLiteException {
-        if (maxDepth == 0) {
+        if (maxDepth == 0)
             maxDepth = getMaxRevTreeDepth();
-        }
+
+        Log.e(TAG, "CBLDatabase: Pruning revisions to max depth %d...", maxDepth);
```

Comment:
This shouldn't be an error log I guessed. Also, `CBLDatabase` is an iOS artifact, could you please change it to something else or remove it?